test/test_managercli.py

```diff
@@ -1525,6 +1525,116 @@ def test_release_unset_updates_repos(self):
                 mock_repo_invoker.update.assert_called_with()
 
 
+class TestRoleCommand(TestCliCommand):
+    command_class = managercli.RoleCommand
+
+    def setUp(self):
+        super(TestRoleCommand, self).setUp(False)
+        self.cc = self.command_class()
+
+    def test_wrong_options_syspurpose_role(self):
+        """It is possible to use --set or --unset options. It's not possible to use both of them together."""
+        try:
+            self.cc.main(["--set", "Foo", "--unset"])
+            self.cc._validate_options()
+        except SystemExit as e:
+            self.assertEqual(e.code, os.EX_USAGE)
+
+    @patch("subscription_manager.managercli.SyspurposeStore")
```

Comment:
Maybe it would be better if we reference the store through the syspurposelib here too (e.g. `syspurposelib.SyspurposeStore`). But then again they are unit tests so we always have access to the the syspurpose stuff anyway.